node/src/chain/acceptor.rs

```diff
@@ -48,6 +48,8 @@ use crate::database::rocksdb::{
 use crate::database::{self, ConsensusStorage, Ledger, Mempool, Metadata};
 use crate::{vm, Message, Network};
 
+use rusk_recovery_tools::state::DUSK_CONSENSUS_KEY;
```

Comment:
Do you mind to "pub use" it in `node/src/lib.rs` and use it here?
So to remove the additional declaration in `rusk/src/lib/node/rusk.rs`?

Reply:
Done